plugins/base/src/test/kotlin/multiplatform/BasicMultiplatformTest.kt

```diff
@@ -25,6 +25,44 @@ class BasicMultiplatformTest : BaseAbstractTest() {
         }
     }
 
+    @Test
+    fun dataTestLongSparseArray() {
+        val testDataDir = getTestDataDir("multiplatform/longSparseArray").toAbsolutePath()
+
+        val configuration = dokkaConfiguration {
+            sourceSets {
```

Comment:
If you use `expect`/`actual` declarations, you need to put dependency on a common source set like `dependentSourceSets = setOf(common.value.sourceSetID)`.

The full working code:
```
                val common =  sourceSet {
                    name = "common"
                    displayName = "common"
                    analysisPlatform = "common"
                    sourceRoots = listOf("$testDataDir/commonMain/")
                }
                sourceSet {
                    name = "jvm"
                    displayName = "JVM"
                    sourceRoots = listOf("$testDataDir/jvmMain/")
                    dependentSourceSets = setOf(common.value.sourceSetID)
                }
                sourceSet {
                    name = "native"
                    displayName = "Native"
                    analysisPlatform = "native"
                    sourceRoots = listOf("$testDataDir/nativeMain/")
                    dependentSourceSets = setOf(common.value.sourceSetID)
                }
```